frontend/front/src/pages/Public/Components/LoginPublicView.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
```

Comment:
I don't think this pop up is necessary. I think just linking it to the login page is sufficient. We're duplicating work and having login to be a popup doesn't look good.

Reply:
I agree, the pop-up is unnecessary. Maybe we just need a button on the login page to return to the home page.